tests/cache/tests.py

```diff
@@ -1473,6 +1474,14 @@ def test_empty_cache_file_considered_expired(self):
             self.assertIs(cache._is_expired(fh), True)
 
 
+class FileBasedCachePathLibTests(FileBasedCacheTests):
+    def setUp(self):
+        super().setUp()
+        for cache_params in settings.CACHES.values():
+            cache_params.update({'LOCATION': Path(self.dirname)})
+        setting_changed.send(self.__class__, setting='CACHES', enter=False)
```

Comment:
Do the changes to the settings need to be reverted in `tearDown()`?

Reply:
Nope. This is already handled by the `override_settings` on the parent class.